corehq/apps/export/models/new.py

```diff
@@ -844,6 +844,13 @@ def get_payload(self, stream=False):
         """
         return self.fetch_attachment(DAILY_SAVED_EXPORT_ATTACHMENT_NAME, stream=stream)
 
+    def copy_export(self):
+        export_json = self.to_json()
+        del export_json['_id']
+        export_json['name'] = '{} - Copy'.format(self.name)
+        new_export = self.__class__.wrap(export_json)
+        new_export.save()
```

Comment:
nit: our architecture guide prefers returning the object and allowing for the client to save the model: https://github.com/dimagi/architecture-guide/blob/master/let_client_save_model.md#best-practice-let-the-client-code-save-the-model